src/types.ts

```diff
@@ -431,3 +431,22 @@ export type Reload = (
   optHref: DOMString | null | undefined,
   opts: HvComponentOptions,
 ) => void;
+
+export type LogPayload = {
```

Comment:
I think that `data`, `prefix`, `source` and `timestamp` are elements of the log message that can remain discreet in the logger implementation, so don't need to be part of the signature exposed by Hyperview

Reply:
You're suggesting we just export "any" and it's up to the logger to interpret the type? I thought it would be easier to be explicit.